open_humans/views.py

```diff
@@ -695,53 +701,54 @@ def get_inbound_connections():
                 .order_by('name')
                 .annotate(count=Count('accesstoken__user', distinct=True)))
 
-    @staticmethod
-    def get_files(is_public):
-        files = (PublicDataAccess.objects
-                 .filter(is_public=is_public)
-                 .values('data_file_model__app_label')
-                 .annotate(count=Count('data_file_model__app_label')))
-
-        for f in files:
-            app_label = f.pop('data_file_model__app_label')
-            app = app_from_label(app_label)
-
-            f['app'] = app.verbose_name
-
-        # sort here instead of in the database since we need to look up the
-        # Django app name
-        return sorted(files, key=itemgetter('app'))
+    def get_source_connections(self):
+        source_connections = dict()
```

Comment:
Hadn't realized there was a speed difference! Microscopic, of course. Sadly not what is causing this code to be slow as hell. I'm sure that "iterating over each UserData, each time performing a filter on PublicDataAccess objects" was not a brilliant approach. Surely a better solution exists, but at this point I just wanted to fix the page - albeit incompletely - so I could move on.